tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -339,4 +339,13 @@ Please read up on general k8s to get an idea of correct formatting for:
         }
     }
 }
+```
+## SSHTests
+sshTests runs through the list of sshTests defined in ssh.go. CheckCPU verifies that the cpu usage of a node does not go past 100%
+Currently does not support Azure and Windows node driver clusters
+
+```json
+"clusterSSHTests": {
```

Comment:
we aren't looking up this name anywhere, so I don't think this would actually run the test right now.